nasa/harmony
# Harmony 1269: Scrape k8s services instead of pods for queue length metrics

## Jira Issue ID
HARMONY-1269

## Description
Changes the prometheus metrics logic to scrape k8s services instead of pods for queue length metrics.

## Local Test Steps
- In app/service-backends/metrics.ts in `getEligibleWorkItemCountForServiceID`, add: `console.log('get harmony metric ' + req.query.serviceID + ' ' + (new Date()).toISOString());`
- `npm install` in the root directory and for service-runner
- `bin/restart-services`
- `kubectl create ns monitoring --kubeconfig ~/.kube/docker-desktop`
- `bin/deploy-hpa`
- start harmony
- Submit a larger request (200 granules) http://localhost:3000/C1233800302-EEDTEST/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?maxResults=200
- Check the logs to see that metrics are being queried every 15s per service, regardless of how many pods have scaled up

## PR Acceptance Checklist
